gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -120,16 +120,16 @@ export const GalleryExampleTile: FunctionComponent<GalleryExampleTileProps> =
           }
         </NxTile.Content>
 
-        { collapseCodeExample ?
+        { uncollapseCodeExample ?
+          <NxTile.Content>{codeExampleElements}</NxTile.Content> :
```

Comment:
Is there a reason to still have the option to uncollapse?

Reply:
> Is there a reason to still have the option to uncollapse?

Not really now that I think about it. Removed here: 67a94c06376d861325202cdcc7e1b5b93aec5bb9